packages/chat-component/src/main.ts

```diff
@@ -133,6 +141,26 @@ export class ChatComponent extends LitElement {
 
   static override styles = [mainStyle];
 
+  // The following block is only necessary when you want to override the component from settings in the outside.
+  // Remove this block when not needed, considering that updated() is a LitElement lifecycle method
+  // that may be used by other components if you update this code.
+  override updated(changedProperties: Map<string | number | symbol, unknown>) {
```

Comment:
check out and see if styleMap directive for lit would be useful here. https://lit.dev/docs/components/styles/#dynamic-classes-and-styles